test/test_schema.py

```diff
@@ -42,9 +29,14 @@ def test_returning_clause(self):
 
     def test_get_indexes(self):
         with testing.db.begin() as conn:
+            conn.execute("CREATE TABLE three_columns (id1 INT, id2 INT, id3 INT)")
+            conn.execute("CREATE INDEX three_columns_idx ON three_columns(id2) INCLUDE(id1)")
+
             insp = inspect(testing.db)
-            indexes = insp.get_indexes("users")
+            indexes = insp.get_indexes("three_columns")
+
+            assert len(indexes) == 1
+            assert indexes[0]["name"] == "three_columns_idx"
+            assert indexes[0]["column_names"] == ["id2", "id1"]
```

Comment:
Can we get pk of this index?

Reply:
Now we can't. We have the ability to add and use `indnkeyatts` column in `pg_index`. We can support getting pk when we need it.

Reply:
Now we can't. We have the ability to add and use `indnkeyatts` column in `pg_index`. The value of `indnkeyatts` means that how many keys in front of `indkey` are pk. We can support getting pk when we need it.